src/components/PaymentLog.vue

```diff
@@ -936,6 +1038,30 @@ export default {
         this.rPayment = rePayment;
         this.pPrice = total;
         this.test1 = false;
+        const months = this.rDuration / 30;
+
+        const cycle = Math.ceil(28 / this.repaymentCircle);
+        const additionalRepayment = this.rPayment / (months * cycle);
+
+        if (
+          (this.selectedProduct.price > 80000 &&
```

Comment:
Isn't this same logic we have here `watchCashPrice`?

Reply:
no the watchCashPrice is checking whether to display additional downpayment or not, this logic here is to determine how much repayment to be added, whether one or two or half,